src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -78,8 +94,30 @@ export class OverflowMenuOption implements AfterViewInit {
 	 * disable/enable interactions
 	 */
 	@Input() disabled = false;
-
+	/**
+	 * If it's an anchor, this is its location
+	 */
 	@Input() href: string;
+	/**
+	 * Allows to add a target to the anchor
+	 */
+	@Input() set target(value: Target) {
+		if (!Object.keys(Target).includes(value)) {
+			return;
```

Comment:
```suggestion
			throw new Error(`target must have one of the following values: ${Object.values(Target).join(', ')}. Please use the Target ENUM exported by carbon-components-angular`);
```

Reply:
I don't think the validation is needed -- users may not understand why their value (even though it's incorrect) is not being saved. If you keep it I'll recommend either throwing an error or logging an error to the console so the user has a chance to be able to debug the issue -- instead of a silent return.

Reply:
Well, it's kind of how HTML works innit? Wrong tags and/or properties get silently ignored. That was my motivation. But i totally understand your point.

Reply:
💯 Definitely how HTML works. But doesn't mean it can't be made to be better. Perhaps just a `console.error()` message instead of an actual error would be more helpful to a user while not breaking things entirely.

Reply:
Got it. Will add the suggestion. Thanks!